### PR TITLE
update alpha dependencies to alpha

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: .github/renovate.json5 [hunk 1]
```diff
@@ -12,16 +12,11 @@
...
-        "io.opentelemetry.proto:opentelemetry-proto",
-        "io.opentelemetry.semconv:opentelemetry-semconv"
-      ],
       // Renovate's default behavior is only to update from unstable -> unstable if it's for the
       // major.minor.patch, under the assumption that you would want to update to the stable version
       // of that release instead of the unstable version for a future release
-      // (TODO remove once the artifacts above release stable versions)
-      "ignoreUnstable": false
+      "matchCurrentVersion": "/^.*-alpha$/",
+      "allowedVersions": "/^.*-alpha$/"
```
Comment (trask) [inline .github/renovate.json5:19]: will this also auto-update from `-alpha` to a stable version?

**Comment by laurit** [comment]
@zeitlinger did you test this or do you guess that this should work?

File: .github/renovate.json5 (hunk 1 above)
Comment (zeitlinger) [reply .github/renovate.json5:19]: no, `allowedVersions` describes which versions are allowed as new versions, which is the regex `^.*-alpha$`

**Comment by laurit** [comment]
> I did not test it, but it should according to https://docs.renovatebot.com/configuration-options/#allowedversions

To reduce the blast radius what do you think of keeping the `matchPackageNames` rules. Instead of matching against `-alpha` we could match agains not snapshot like in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/bc53f2ead172e69e44f48ee2446465d4f782d1b2/.github/renovate.json5#L65 I wouldn't be surprised if `"ignoreUnstable": false` is still needed.

**Pull request closed without merging**